tbx/static_src/sass/components/_pagination.scss

```diff
@@ -14,12 +14,12 @@
     }
 
     &__page {
-        @include font-size('heading-six');
+        @include font-size('heading-five');
         color: var(--color--heading);
         display: block;
 
         @include media-query(large) {
-            @include font-size('heading-five');
+            @include font-size('heading-four');
```

Comment:
It's on the verbose side, but you could use `font-size: nth(map-get($large-font-sizes, 'heading-four'), 1);` here to access the font size (and would need the same for line-height) so not to include the CSS for mobile for 'heading-four' when it isn't used.

![image](https://github.com/torchbox/torchbox.com/assets/31627284/52593285-2bbe-4193-a3bb-8ac649940163)

Reply:
Updated in 6088e413a9b8d7790712ee88330694f885824372